lib/channel/Player.ts

```diff
@@ -23,7 +23,9 @@ export class Player extends Base {
   private __fetchTimeoutId:   number = 0;
   private __audioManager:     AudioManager;
   private __started:          boolean;
+  private __playlist:         Playlist = null;
   private __clock?:           SyncClock = null;
+  private __isFetchingRunning:boolean = false;
```

Comment:
@konrad-marzec naming is not consequent, for being started we just use `__started` not `isStarted`, please change to `__fetching`